legal-api/src/legal_api/services/digital_credentials.py

```diff
@@ -29,131 +30,184 @@ class DigitalCredentialsService:
 
     business_schema = {
         'attributes': [
-            'legalName',
-            'foundingDate',
-            'taxId',
-            'homeJurisdiction',
-            'legalType',
-            'identifier'
+            'business_name',
+            'company_status',
+            'credential_id',
+            'identifier',
+            'registered_on_dateint',
+            'role',
+            'cra_business_number',
+            'family_name',
+            'business_type',
+            'given_names',
         ],
-        'schema_name': 'business_schema',  # do not change schema name. this is the name registered in aca-py agent
-        'schema_version': '1.0.0'  # if attributes changes update schema_version to re-register
+        # do not change schema name. this is the name registered in aca-py agent
+        'schema_name': 'digital_business_card',
+        # if attributes change update schema_version to re-register
+        'schema_version': '1.0.0'
```

Comment:
so these values are good as is?  as in we don't need to have this pulled from config(`BUSINESS_SCHEMA_NAME`, `BUSINESS_SCHEMA_VERSION`)  or anything?

Reply:
Personally I would get them from the config. If the name or version ever gets updated no code needs to be updated. I can fix that.